sharc/parameters/parameters_rns.py

```diff
@@ -45,6 +46,8 @@ class ParametersRns(ParametersBase):
     season: str = "SUMMER"
     # Adjacent channel selectivity [dB]
     acs: float = 30.0
+    # Parameters for P528 model
+    ParametersP528 = ParametersP528()
```

Comment:
You're accessing `param_p528` on station_factory, and setting `param_p528` on yaml files.
You should change the name here to `param_p528` as well:
```suggestion
    param_p528 = ParametersP528()
```